.github/workflows/scheduled.yml

```
@@ -396,3 +397,80 @@ jobs:
           name: join-fuzzer-failure-artifacts
           path: |
             /tmp/join_fuzzer_repro
+
+
+  presto-java-aggregation-fuzzer-run:
+    runs-on: 8-core
```

```suggestion
    name: Presto Java Fuzzer
    runs-on: 8-core
```
(or whatever you want to call it) 
Also it's still in `experimental.yml` with the addition here and can be removed from that?